README.md

```diff
@@ -51,13 +51,19 @@ In this section we demonstrate a minimal example of sampling from a multivariate
   
 ```julia
 using AdvancedHMC, Distributions, ForwardDiff
+using LogDensityProblems
 using LinearAlgebra
 
+# Define the target distribution using the `LogDensityProblem` interface
+struct LogTargetDensity
+    dim::Int
+end
+LogDensityProblems.logdensity(p::LogTargetDensity, θ) = logpdf(MvNormal(zeros(p.dim), I), θ)
```

Comment:
More efficient alternatives would be
```suggestion
LogDensityProblems.logdensity(p::LogTargetDensity, θ) = logpdf(MvNormal(Zeros(p.dim), I), θ)
```
or
```suggestion
LogDensityProblems.logdensity(p::LogTargetDensity, θ) = logpdf(MvNormal(Eye(p.dim)), θ)
```
Both require `using FillArrays`.

Reply:
Am aware of this but do we want to the introduce this additional "complexity"? I feel like if a user is familiar with FillArrays.jl, they'll immediately know to use that implementation instead, while if you're not then maybe this is not the place to familiarize yourself with it?

Reply:
Hmm maybe you're right. On the other hand, maybe the README would be a good opportunity to teach people that don't know these more efficient implementations.

Reply:
> On the other hand, maybe the README would be a good opportunity to teach people that don't know these more efficient implementations.

I don't think that should be a goal of this README. In fact, wouldn't it be simpler to just drop Distributions altogether and just use 
```suggestion
LogDensityProblems.logdensity(p::LogTargetDensity, θ) = -sum(abs2, θ) / 2  # standard multivariate normal
```
?

Reply:
> I don't think that should be a goal of this README.

Exactly, and in particular with `using` and exported functionality, a user might not even be able to identify what's from which package. 

> In fact, wouldn't it be simpler to just drop Distributions altogether and just use

Was thinking the same, but figured I'd leave it as is in an attempt to avoid this such a discussion to clog the PR :sweat_smile: But I'm happy to make that change if we want. @xukai92 you okay with that?

Reply:
I made the change suggested by @sethaxen